scene/gui/tree.cpp

```diff
@@ -1370,6 +1384,8 @@ void TreeItem::set_custom_as_button(int p_column, bool p_button) {
 
```

Comment:
Missing check for value being different.

Reply:
Will add 🙂